--- glabels-3.4.1/src/template-history.h.orig	2016-03-04 05:01:15.000000000 +0100
+++ glabels-3.4.1/src/template-history.h	2020-09-27 14:43:16.510439592 +0200
@@ -28,7 +28,7 @@
 G_BEGIN_DECLS
 
 
-glTemplateHistoryModel *gl_template_history;
+extern glTemplateHistoryModel *gl_template_history;
 
 
 void            gl_template_history_init (void);
--- glabels-3.4.1/src/font-history.h.orig	2016-03-04 05:01:15.000000000 +0100
+++ glabels-3.4.1/src/font-history.h	2020-09-27 14:43:46.635333431 +0200
@@ -28,7 +28,7 @@
 G_BEGIN_DECLS
 
 
-glFontHistoryModel *gl_font_history;
+extern glFontHistoryModel *gl_font_history;
 
 
 void            gl_font_history_init (void);
