diff -aurN glabels-1.92.1.orig/data/glabels.desktop.in glabels-1.92.1/data/glabels.desktop.in
--- glabels-1.92.1.orig/data/glabels.desktop.in	2003-10-19 04:21:11.000000000 +0200
+++ glabels-1.92.1/data/glabels.desktop.in	2003-10-28 15:14:31.038533088 +0100
@@ -1,7 +1,9 @@
 [Desktop Entry]
 Name=gLabels Label Designer
+Name[pl]=gLabels
 Name[ja]=gLabels
 Comment=Create labels, business cards and media covers.
+Comment[pl]=Tworzy etykiety i wizytówki
 Comment[ja]=¥é¥Ù¥ë¤È¥Ó¥¸¥Í¥¹¥«¡¼¥É¤ÎºîÀ®¥×¥í¥°¥é¥à
 Exec=glabels
 Icon=glabels/glabels-icon.png
