diff -urN glabels-2.0.2.orig/data/desktop/glabels.desktop.in glabels-2.0.2/data/desktop/glabels.desktop.in
--- glabels-2.0.2.orig/data/desktop/glabels.desktop.in	2005-01-25 12:57:52.559786240 +0100
+++ glabels-2.0.2/data/desktop/glabels.desktop.in	2005-01-25 13:00:19.882389800 +0100
@@ -6,6 +6,6 @@
 Icon=glabels.png
 Terminal=false
 Type=Application
-Categories=GNOME;Application;Office;
+Categories=GTK;GNOME;Office;
 StartupNotify=true
 MimeType=application/x-glabels;
