diff -aurN glabels-1.91.20030513.orig/configure.in glabels-1.91.20030513/configure.in
--- glabels-1.91.20030513.orig/configure.in	Sun Apr 27 05:55:45 2003
+++ glabels-1.91.20030513/configure.in	Tue May 13 14:26:18 2003
@@ -42,8 +42,8 @@
 libgnomeui-2.0 >= $LIBGNOMEUI_REQUIRED \
 libbonobo-2.0 >= $LIBBONOBO_REQUIRED \
 libxml-2.0 >= $LIBXML_REQUIRED \
-libgnomeprint-2.0 >= $LIBGNOMEPRINT_REQUIRED \
-libgnomeprintui-2.0 >= $LIBGNOMEPRINTUI_REQUIRED \
+libgnomeprint-2.2 >= $LIBGNOMEPRINT_REQUIRED \
+libgnomeprintui-2.2 >= $LIBGNOMEPRINTUI_REQUIRED \
 libgnomecanvas-2.0 >= $LIBGNOMECANVAS_REQUIRED \
 gdk-pixbuf-2.0 >= $GDKPIXBUF_REQUIRED \
 )
diff -aurN glabels-1.91.20030513.orig/src/glabels-batch.c glabels-1.91.20030513/src/glabels-batch.c
--- glabels-1.91.20030513.orig/src/glabels-batch.c	Tue Dec  3 03:50:13 2002
+++ glabels-1.91.20030513/src/glabels-batch.c	Tue May 13 14:29:40 2003
@@ -23,7 +23,7 @@
 #include <config.h>
 
 #include <gnome.h>
-#include <libgnomeprint/gnome-print-master.h>
+#include <libgnomeprint/gnome-print-job.h>
 
 #include "merge-init.h"
 #include "xml-label.h"
@@ -74,7 +74,7 @@
 	gint               rc;
 	GSList            *p, *file_list = NULL;
 	gint               n_files;
-	GnomePrintMaster  *master = NULL;
+	GnomePrintJob	  *master = NULL;
 	gchar             *abs_fn;
 	GnomePrintConfig  *config = NULL;
 	glLabel           *label = NULL;
@@ -127,7 +127,7 @@
 		if ( status == XML_LABEL_OK ) {
 
 			if ( master == NULL ) {
-				master = gnome_print_master_new ();
+				master = gnome_print_job_new (NULL);
 			}
 
 			gl_print_batch (master, label, n_sheets, n_copies,
@@ -143,11 +143,11 @@
 	if ( master != NULL ) {
 
 		abs_fn = gl_util_make_absolute ( output );
-		gnome_print_master_print_to_file (master, abs_fn);
+		gnome_print_job_print_to_file (master, abs_fn);
 		g_free( abs_fn );
 
-		gnome_print_master_close (master);
-		gnome_print_master_print (master);
+		gnome_print_job_close (master);
+		gnome_print_job_print (master);
 
 		g_object_unref (master);
 	}
diff -aurN glabels-1.91.20030513.orig/src/print-dialog.c glabels-1.91.20030513/src/print-dialog.c
--- glabels-1.91.20030513.orig/src/print-dialog.c	Fri Dec 27 17:55:18 2002
+++ glabels-1.91.20030513/src/print-dialog.c	Tue May 13 15:05:42 2003
@@ -26,10 +26,9 @@
 #include <ctype.h>
 #include <gtk/gtk.h>
 #include <libgnomeprint/gnome-print-paper.h>
-#include <libgnomeprintui/gnome-printer-dialog.h>
 #include <libgnomeprintui/gnome-print-dialog.h>
-#include <libgnomeprint/gnome-print-master.h>
-#include <libgnomeprintui/gnome-print-master-preview.h>
+#include <libgnomeprint/gnome-print-job.h>
+#include <libgnomeprintui/gnome-print-job-preview.h>
 
 #include "print-dialog.h"
 #include "hig.h"
@@ -222,7 +221,7 @@
 	vbox = gl_hig_vbox_new (GL_HIG_VBOX_OUTER);
 
 	printer_select =
-		gnome_printer_selection_new (gnome_print_config_default ());
+		gnome_printer_selector_new (gnome_print_config_default ());
 	gtk_widget_show (printer_select);
 	gl_hig_vbox_add_widget (GL_HIG_VBOX(vbox), printer_select);
 
@@ -258,7 +257,7 @@
 		printer_select = g_object_get_data (G_OBJECT(dlg),
 						    "printer_select");
 
-		config = gnome_printer_selection_get_config (GNOME_PRINTER_SELECTION(printer_select));
+		config = gnome_printer_selector_get_config (printer_select);
 
 		outline_flag =
 			gtk_toggle_button_get_active (GTK_TOGGLE_BUTTON
@@ -314,19 +313,19 @@
 	      gboolean          outline_flag,
 	      gboolean          reverse_flag)
 {
-	GnomePrintMaster *master;
+	GnomePrintJob *master;
 
-	master = gnome_print_master_new_from_config (config);
+	master = gnome_print_job_new (config);
 	gl_print_simple (master, label, n_sheets, first, last,
 			 outline_flag, reverse_flag);
-	gnome_print_master_close (master);
+	gnome_print_job_close (master);
 
 	if (preview_flag) {
 		GtkWidget *preview_widget =
-		    gnome_print_master_preview_new (master, _("Print preview"));
+		    gnome_print_job_preview_new (master, _("Print preview"));
 		gtk_widget_show (GTK_WIDGET (preview_widget));
 	} else {
-		gnome_print_master_print (master);
+		gnome_print_job_print (master);
 	}
 
 	g_object_unref (G_OBJECT (master));
@@ -345,9 +344,9 @@
 		    gboolean          outline_flag,
 		    gboolean          reverse_flag)
 {
-	GnomePrintMaster *master;
+	GnomePrintJob *master;
 
-	master = gnome_print_master_new_from_config (config);
+	master = gnome_print_job_new (config);
 	if ( collate_flag ) {
 		gl_print_merge_collated (master, label,
 					 n_copies, first,
@@ -357,14 +356,14 @@
 					   n_copies, first,
 					   outline_flag, reverse_flag);
 	}
-	gnome_print_master_close (master);
+	gnome_print_job_close (master);
 
 	if (preview_flag) {
 	        GtkWidget *preview_widget =
-		    gnome_print_master_preview_new (master, _("Print preview"));
+		    gnome_print_job_preview_new (master, _("Print preview"));
 		gtk_widget_show (GTK_WIDGET (preview_widget));
 	} else {
-		gnome_print_master_print (master);
+		gnome_print_job_print (master);
 	}
 
 	g_object_unref (G_OBJECT (master));
diff -aurN glabels-1.91.20030513.orig/src/print.c glabels-1.91.20030513/src/print.c
--- glabels-1.91.20030513.orig/src/print.c	Sat Jan 25 01:43:45 2003
+++ glabels-1.91.20030513/src/print.c	Tue May 13 14:33:19 2003
@@ -26,7 +26,7 @@
 #include <ctype.h>
 #include <gtk/gtk.h>
 #include <libgnomeprint/gnome-print-paper.h>
-#include <libgnomeprintui/gnome-printer-dialog.h>
+#include <libgnomeprintui/gnome-print-dialog.h>
 
 #include "print.h"
 #include "label.h"
@@ -74,7 +74,7 @@
 /*=========================================================================*/
 /* Private function prototypes.                                            */
 /*=========================================================================*/
-static PrintInfo *print_info_new              (GnomePrintMaster *master,
+static PrintInfo *print_info_new              (GnomePrintJob *master,
 					       glLabel          *label);
 
 static void       print_info_free             (PrintInfo       **pi);
@@ -162,7 +162,7 @@
 /* Simple (no merge data) print command.                                     */
 /*****************************************************************************/
 void
-gl_print_simple (GnomePrintMaster *master,
+gl_print_simple (GnomePrintJob *master,
 		 glLabel          *label,
 		 gint              n_sheets,
 		 gint              first,
@@ -206,7 +206,7 @@
 /* Merge print command (collated copies)                                     */
 /*****************************************************************************/
 void
-gl_print_merge_collated (GnomePrintMaster *master,
+gl_print_merge_collated (GnomePrintJob *master,
 			 glLabel          *label,
 			 gint              n_copies,
 			 gint              first,
@@ -274,7 +274,7 @@
 /* Merge print command (uncollated copies)                                   */
 /*****************************************************************************/
 void
-gl_print_merge_uncollated (GnomePrintMaster *master,
+gl_print_merge_uncollated (GnomePrintJob *master,
 			   glLabel          *label,
 			   gint              n_copies,
 			   gint              first,
@@ -344,7 +344,7 @@
 /* Batch print.  Call appropriate function above.                            */
 /*****************************************************************************/
 void
-gl_print_batch (GnomePrintMaster *master,
+gl_print_batch (GnomePrintJob *master,
 		glLabel          *label,
 		gint              n_sheets,
 		gint              n_copies,
@@ -380,7 +380,7 @@
 /* PRIVATE.  new print info structure                                        */
 /*---------------------------------------------------------------------------*/
 static PrintInfo *
-print_info_new (GnomePrintMaster *master,
+print_info_new (GnomePrintJob *master,
 		glLabel          *label)
 {
 	PrintInfo            *pi = g_new0 (PrintInfo, 1);
@@ -395,8 +395,8 @@
 		return NULL;
 	}
 
-	pi->pc = gnome_print_master_get_context (master);
-	pi->config = gnome_print_master_get_config (master);
+	pi->pc = gnome_print_job_get_context (master);
+	pi->config = gnome_print_job_get_config (master);
 
 	if ((template != NULL) && (template->page_size != NULL)) {
 
diff -aurN glabels-1.91.20030513.orig/src/print.h glabels-1.91.20030513/src/print.h
--- glabels-1.91.20030513.orig/src/print.h	Sun Dec  8 07:31:57 2002
+++ glabels-1.91.20030513/src/print.h	Tue May 13 14:39:40 2003
@@ -23,13 +23,13 @@
 #define __PRINT_H__
 
 #include <gnome.h>
-#include <libgnomeprint/gnome-print-master.h>
-#include <libgnomeprintui/gnome-print-master-preview.h>
+#include <libgnomeprint/gnome-print-job.h>
+#include <libgnomeprintui/gnome-print-job-preview.h>
 #include "label.h"
 
 G_BEGIN_DECLS
 
-void gl_print_simple           (GnomePrintMaster *master,
+void gl_print_simple           (GnomePrintJob *master,
 				glLabel          *label,
 				gint              n_sheets,
 				gint              first,
@@ -37,21 +37,21 @@
 				gboolean          outline_flag,
 				gboolean          reverse_flag);
 
-void gl_print_merge_collated   (GnomePrintMaster *master,
+void gl_print_merge_collated   (GnomePrintJob *master,
 				glLabel          *label,
 				gint              n_copies,
 				gint              first,
 				gboolean          outline_flag,
 				gboolean          reverse_flag);
 
-void gl_print_merge_uncollated (GnomePrintMaster *master,
+void gl_print_merge_uncollated (GnomePrintJob *master,
 				glLabel          *label,
 				gint              n_copies,
 				gint              first,
 				gboolean          outline_flag,
 				gboolean          reverse_flag);
 
-void gl_print_batch            (GnomePrintMaster *master,
+void gl_print_batch            (GnomePrintJob *master,
 				glLabel          *label,
 				gint              n_sheets,
 				gint              n_bcopies,
